database/database.go

```diff
@@ -14,15 +14,16 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 	"go.mongodb.org/mongo-driver/mongo/readpref"
 	"go.mongodb.org/mongo-driver/mongo/writeconcern"
+	"go.sia.tech/siad/crypto"
 )
 
 const (
 	// MongoDefaultTimeout is the timeout for the context used in testing
 	// whenever a context is sent to mongo
-	MongoDefaultTimeout = time.Minute
+	MongoDefaultTimeout = 10 * time.Minute
 
 	// mongoIndexCreateTimeout is the timeout used when creating indices
-	mongoIndexCreateTimeout = 10 * time.Second
+	mongoIndexCreateTimeout = 10 * time.Minute
```

Comment:
Aren't both of these exorbitantly high? If creating indexes doesn't finish within a couple of seconds then there is something very wrong and we should error out, rather than wait for 10 minutes before the container fails. Adding such a long timeout will only delay our problem investigation.

Reply:
I don't mind lowering it but on production it blocks on creating indices in the background and I've seen it take more than one minute... We already know we have a problem with our mongo cluster right, we just haven't gotten to it yet. I'd rather have the index eventually then a failed container.

I guess we can lower it to one minute.